even_fib_nums.js

```diff
@@ -5,31 +5,50 @@
  * @param  {Number} maxFibValue
  * @return {Number} sum
  */
-function _sumFibs( maxFibValue ) {
+function _sumFibs(maxFibValue) {
   var sum = 0;
-
-  // do your work here
+console.log(maxFibValue)
+  var arr = [1, 2];
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[arr.length - 1] < maxFibValue) {
+      arr.push(arr[arr.length - 1] + arr[arr.length - 2]);
+    }
+  }
+  for (let j = 0; j < arr.length; j++) {
+    if (arr[j] % 2 === 0) {
+      sum += arr[j];
+    }
+  }
 
   return sum;
 }
 
 // bonus round
-function _highestFibonacciNumber (maxFibValue){
-  var highest = 0;
+function _highestFibonacciNumber(maxFibValue) {
 
-  //define your base case, validate your input
+  var highest = 0;
+  var arr = [1, 2];
+  for (let i = 0; i < arr.length; i++) {
```

Comment:
Same note as above in regard to using two loops. Maybe figure out how to use one loop while keeping track of the highest fibonacci number at the same time?